Modules/IO/ImageBase/test/itkImageSeriesReaderSamplingTest.cxx

```diff
@@ -58,6 +58,18 @@ itkImageSeriesReaderSamplingTest(int ac, char * av[])
       return EXIT_FAILURE;
     }
 
+    double globalSamplingDeviation = 0.0;
+    if (itk::ExposeMetaData<double>(
+          reader->GetOutput()->GetMetaDataDictionary(), "ITK_non_uniform_sampling_deviation", globalSamplingDeviation))
+    {
+      std::cout << "output ITK_non_uniform_sampling_deviation = " << globalSamplingDeviation << std::endl;
```

Comment:
What is the output of this in the test?

Reply:
it depends on the deviation , the test checks that the flag is present.

Reply:
I meant, what does [this test](https://github.com/InsightSoftwareConsortium/ITK/blob/e79b70f13afafa2eb0ef48f6ac90af185379080b/Modules/IO/ImageBase/test/CMakeLists.txt#L282-L287) produce?

Reply:
it simulates situation when there are extra slices , artificially creating non-uniform sampling.

Reply:
I meant, what is the output of this line in [this existing test](https://github.com/InsightSoftwareConsortium/ITK/blob/e79b70f13afafa2eb0ef48f6ac90af185379080b/Modules/IO/ImageBase/test/CMakeLists.txt#L282-L287)? Half of the regular spacing perhaps? But it was a question out of curiosity, it does not matter much.

Reply:
well, it detects nonuniform sampling , since it there are two slices glued together - it detects maximum deviation of 1.06 - which is expected slice distance when 4 slices are present. 

This test is a little artificial, usually there are missing slices and not duplicated slices.